packages/metricscollector/spec

```diff
@@ -36,6 +36,7 @@ files:
 - app-autoscaler/src/github.com/gogo/protobuf/gogoproto/*.go # gosub
 - app-autoscaler/src/github.com/gogo/protobuf/proto/*.go # gosub
 - app-autoscaler/src/github.com/gogo/protobuf/protoc-gen-gogo/descriptor/*.go # gosub
+- app-autoscaler/src/github.com/gorilla/context/*.go # gosub
```

Comment:
I was wondering we need this package for dumping up pruner module. app-autoscaler has no changes for metricscollector, eventgenerator and scalingengine.